awx-ee/_build/requirements.txt

```diff
@@ -1,8 +1,21 @@
+git+https://github.com/hpi-schul-cloud/infra-tools.git#subdirectory=onepwd
+ansible==8.0.0
+ansible-core==2.15.2
+molecule==3.3.4
+kubernetes==17.17.0
+ionoscloud==6.1.3
+pyotp==2.6.0
+grafana_api
+url64==0.1.0
+boto3==1.21.43
+pexpect==4.8.0
+jinja2==3.0.3
+urllib3~=1.26
+pyOpenSSL==23.1.1
+dnspython==2.3.0
+
 awxkit>=19.4.0
-cryptography >= 3.0
-ipaddress
-jsonpatch
-kubernetes>= 12.0.0
-PyYAML >= 3.11
-requests >= 1.1
-urllib3
\ No newline at end of file
+cryptography>=39.0.0
+ipaddress>=1.0.0
+jsonpatch>=1.32
+requests>=2.31.0
```

Comment:
Had to put the python requirements in here, since they have to get installed as root. Once the Docker Image is ready it will run as user 1000 and all python packages installed as user 1000 will be installed in the users python site packages. However Ansible can only see the packages that are installed in the root site packages.